### PR TITLE
refactor to use async/await when it needed only

### PR DESCRIPTION
**Pull request by vesalink**

### CODE REVIEW

**Review by mbystedt — CHANGES_REQUESTED** [review]
Let's have a session discussing the changes.

File: lib/Jira.js [hunk 1]
```diff
@@ -281,11 +282,10 @@ class Jira {
      * @param {*} issueKey issueKey to get the issue.
      * @returns {Promise} found issue.
      */
-    async getIssue(issueKey) {
+    getIssue(issueKey) {
         try {
-            const issue = await this.getJiraClient().issue.getIssue({ issueKey: issueKey })
+            return this.getJiraClient().issue.getIssue({ issueKey: issueKey })
```
Comment (mbystedt) [inline lib/Jira.js:287]: Nice change!

File: lib/dbBackupRecovery.js [hunk 2]
```diff
@@ -7,13 +7,13 @@ const config = require(`${process.cwd()}/lib/config.js`)
 const { childProcess } = require('./util-functions')
 const CONST = require('./constants')
 
-;(async function() {
+;(function() {
```
Comment (mbystedt) [inline lib/dbBackupRecovery.js]: Maybe remove the function()?

**Review by mbystedt — DISMISSED** [review]
Looking good. The only things I saw was that lib/dbBackupRecovery.js is unnecessarily wrapping the code in a function.

**Comment by vesalink** [comment]
> Looking good. The only things I saw was that lib/dbBackupRecovery.js is unnecessarily wrapping the code in a function.

done

**Review by mbystedt — APPROVED** [review]

**Pull request merged**